packages/calcite-components/src/components/carousel/carousel.tsx

```diff
@@ -124,7 +124,7 @@ export class Carousel extends LitElement implements InteractiveComponent {
   /** Specifies how and if the "previous" and "next" arrows are displayed. */
   @property({ reflect: true }) arrowType: ArrowType = "inline";
 
-  /** When `true`, the carousel will autoplay and controls will be displayed. When "paused" at time of initialization, the carousel will not autoplay, but controls will be displayed. */
+  /** When present, the carousel will autoplay and controls will be displayed. When "paused" at time of initialization, the carousel will not autoplay, but controls will be displayed. */
   @property({ reflect: true }) autoplay: AutoplayType = false;
 
   /** When `autoplay` is `true`, specifies in milliseconds the length of time to display each Carousel Item. */
```

Comment:
I missed this as well, but this should also be updated.

```suggestion
  /** When `autoplay` is present, specifies in milliseconds the length of time to display each Carousel Item. */
```

Reply:
Good catch 🐟